superset-frontend/src/views/CRUD/hooks.ts

```diff
@@ -210,7 +210,7 @@ export function useSingleViewResource<D extends object = any>(
             t(
               'An error occurred while fetching %ss: %s',
               resourceLabel,
-              errMsg,
+              JSON.stringify(errMsg),
```

Comment:
I am not sure if that's best way to get better error message. The message has been `[Object, Object]`.

Reply:
This should fix the immediate issue of the error not being displayed, however long term we should probably surface these errors inline in the forms, next to the fields. We could also add validators to the form fields to ensure we're not sending invalid json.